docs/src/content/overview-features.md

```diff
@@ -43,6 +44,81 @@ You may want to use client-side replay in conjunction with the `anticache`
 option, to make sure the server responds with complete data.
 
 
+## Map Local
+
+The `map_local` option lets you specify an arbitrary number of patterns that
+define redirections of HTTP requests to local files or directories.
+The local file is fetched instead of the original resource
+and transparently returned to the client.
+
+`map_local` patterns look like this:
+
+```
+|url-regex|local-path
+|flow-filter|url-regex|local-path
+```
+
+* **local-path** is the file or directory that should be served to the client.
+
+* **url-regex** is a regular expression applied on the request URL. It must match for a redirect to take place.
+
+* **flow-filter** is an optional mitmproxy [filter expression]({{< relref "concepts-filters">}})
+that additionally constrains which requests will be redirected.
+
+### Examples
+
+Pattern | Description
+------- | -----------
+`\|example.com/main.js\|~/main-local.js` | Replace `example.com/main.js` with `~/main-local.js`.
+`\|example.com/static\|~/static` | Replace `example.com/static/foo/bar.css` with `~/static/foo/bar.css`.
+`\|example.com/static/foo\|~/static` | Replace `example.com/static/foo/bar.css` with `~/static/bar.css`.
+`\|~m GET\|example.com/static\|~/static` | Replace `example.com/static/foo/bar.css` with `~/static/foo/bar.css` (but only for GET requests).
+
+### Details
+
+If *local-path* is a file, this file will always be served. File changes will be reflected immediately, there is no caching.
```

Comment:
*immediately* as in: the next time the file is requested?
Or are you actively re-opening the file in the middle of an ongoing transfer?

Reply:
immediately as in the next time it is requested. I thought that'd be sufficiently clear here, isn't it?

Reply:
Maybe the second sentence is unnecessary? Why bother the reader with a behavior (caching) that we _didn't_ implement. It behaves as one would expect and everyone is happy.

Reply:
What happens if the file being served is open in an editor and it gets saved in the middle of a transfer?

Reply:
We just do `file.read_bytes()`, everything that breaks that we don't care about. Which reminds that we should have a test for file read errors. We shouldn't silently break on that and serve a 502/400 instead. :)